README.md

```diff
@@ -41,13 +41,14 @@ cargo build --release
 Purge any existing dev chain state:
 
 ```bash
-./target/release/hack-hydra-dx purge-chain --dev
```

Comment:
It was possible to run solo chain with babe, is this not possible anymore?

Reply:
yes, it works. I've fixed readme in next commit